addons/sale_loyalty/tests/test_loyalty.py

```diff
@@ -79,6 +86,8 @@ def test_nominative_programs(self):
         order._update_programs_and_rewards()
         claimable_rewards = order._get_claimable_rewards()
         self.assertEqual(len(claimable_rewards), 1, "The ewallet program should not be applicable since the card has no points.")
+        vals = order._get_reward_values_discount(loyalty_program.reward_ids[0], loyalty_card)
+        self.assertEqual(vals[0]['points_cost'], 9, "Can only use a entire amount of required points")
```

Comment:
100 chars max =P

Do you mean "an integer amount"/"a whole amount" instead? Then maybe first assert its type with that message. And after that it's value (just because it's never bad to do so)